src/hyperlight_host/src/sandbox/uninitialized_evolve.rs

```diff
@@ -99,13 +101,15 @@ pub(super) fn evolve_impl_multi_use(u_sbox: UninitializedSandbox) -> Result<Mult
 }
 
 #[instrument(err(Debug), skip_all, parent = Span::current(), level = "Trace")]
+#[allow(clippy::too_many_arguments)]
```

Comment:
nit: I think that we'll need to refactor the functions that take too many arguments at some point.